andino_description/urdf/include/andino_caster_macro.urdf.xacro

```diff
@@ -31,10 +31,10 @@
           <origin xyz="0 0 ${-wheel_props['base']['size']['z']/2.0}" rpy="0 0 0" />
           <geometry>
             <xacro:if value="${scale != '' }" >
-              <mesh filename="package://andino_description/meshes/${robot_name}/${wheel_props['base']['mesh']}" scale="${scale}"/>
+              <mesh filename="file://$(find andino_description)/meshes/${robot_name}/${wheel_props['base']['mesh']}" scale="${scale}"/>
```

Comment:
This change isn't ideal. Imagine the following scenario:
1. Launchs robot (real).
2. From another pc(where andino description is also installed), you connect to the network, and proceed to open rviz in order to visualize topics and so. 
3. You won't be able to see the robot model if we apply this change. Why? Well, the `$(find andino_description)` is processed by xacro and replaced by an absolute path, which will be particular to the robot's onboard computer. 

### Context 

This is actually a bug in `gazebo_ros_pkgs` in the `ros2` branch. `package` is parsed mistakenly to `model`

There is already a PR and several issues created presenting this scenario.
https://github.com/ros-simulation/gazebo_ros_pkgs/pull/1284

Reply:
We would need to add a workaround for this, I am thinking of three possible alternatives.
1. We can have an argument in the XACRO file, for indicating which convention to use.
`package://andino_description` vs `file://$(find andino_description)`. By default it should be the former. And we should add a gooood comment there stating why we have this argument.

2. Leave the xacro description with `package` as the original. Then, when processing the xacro in the launch file of gazebo sim we could replace the `package://andino_description` appearance by `file://$(find andino_description)`.
As this is an issue with `gazebo_ros_pkgs` having the workaround directly in the `andino_gazebo` package's launch file and not modifying the `andino_description` sounds better to me.

3. (Please no :D ) Duplicat urdf files to `andino_gazebo` and modify as you wish